pytradfri/resource.py

```diff
@@ -27,7 +30,7 @@ class ApiResource:
     """Base object for resources returned from the gateway."""
 
     _model_class: type[ApiResourceResponse] | None = None
-    raw: TypeRaw | ApiResourceResponse
+    raw: TypeRaw | ApiResourceResponse | GroupResponse
```

Comment:
Since GroupResponse inherits ApiResourceResponse, we don't need to change this.
```suggestion
    raw: TypeRaw | ApiResourceResponse
```